src/components/Items/ProfileContact.js

```diff
@@ -115,7 +115,7 @@ const styles = StyleSheet.create({
   fullname: (isMe) => ({
     fontSize: normalizeFontSize(14),
 
-    color: isMe ? COLORS.signed_primary : COLORS.white2,
+    color: isMe ? COLORS.signed_primary : COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the fullname text has been changed from `COLORS.white2` to `COLORS.white` when `isMe` is false. Ensure that this color change maintains good contrast with the background for readability.
```diff
-    color: isMe ? COLORS.signed_primary : COLORS.white2,
+    color: isMe ? COLORS.signed_primary : COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->